java/fory-core/src/test/java/org/apache/fory/CrossLanguageTest.java

```diff
@@ -798,4 +799,55 @@ public void testEnumField() throws java.io.IOException {
     Assert.assertEquals(xserDe(fory, a), a);
     structRoundBack(fory, a, "test_enum_field");
   }
+
+  @Test
+  public void testCodeGen() {
```

Comment:
Could we split it into two tests, one for SCHEMA_CONSISTENT, and another for COMPATIBLE.

And perhaps we could create a new test file `CodegenXlangTest.java` under `org.apache.fory.xlang` package, and move this test into that file. `CrossLanguageTest` is a little big currently